### ISSUE TITLE
In NVDA, all options are announced while navigating through options list.

### ISSUE THREAD

**Issue by siddharaj-suryavanshi** [JedWatson/react-select#5556]
When tested with NVDA screen reader in Firefox browser on windows 10 OS, the NVDA announces all the options together while navigating through options list via up/down arrow keys. 

All the options together should not be announced by the NVDA screen reader. Instead only currently focused opinion should be announced.

**Comment by zbianca** [comment]
I can reproduce at https://codesandbox.io/s/lm73rw?module=/example.tsx.
NVDA + Chrome is okay.
NVDA + Firefox, on ArrowDown reads all options and then says "expanded". It doesn't inform that option Blue is disabled nor the number of options.

**Comment by halain** [comment]
The same thing happens to me, any update on this?
